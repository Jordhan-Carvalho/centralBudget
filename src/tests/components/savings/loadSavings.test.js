import React from 'react';
import { shallow } from 'enzyme';
import LoadSavings from '../../../components/savings/loadSavings';

test('should render saving page', () => {
    const wrapper = shallow(<LoadSavings />);
    expect(wrapper).toMatchSnapshot();
});