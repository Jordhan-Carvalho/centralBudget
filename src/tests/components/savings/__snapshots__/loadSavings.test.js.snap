// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render saving page 1`] = `
<Fragment>
  <h1>
    Savings List
  </h1>
  <hr />
  <ConnectFunction />
  <ConnectFunction
    type="saving"
  />
  <hr />
  <ConnectFunction />
  <hr />
  <Link
    to="/createsav"
  >
    <button
      className="btn btn-dark"
    >
      Add Saving
    </button>
  </Link>
</Fragment>
`;
